UserListFragment.java

```diff
@@ -0,0 +1,148 @@
+package me.grishka.houseclub.fragments;
+
+import android.app.Activity;
+import android.content.res.Configuration;
+import android.graphics.Bitmap;
+import android.graphics.drawable.ColorDrawable;
+import android.graphics.drawable.Drawable;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.RecyclerView;
+import me.grishka.appkit.Nav;
+import me.grishka.appkit.fragments.BaseRecyclerFragment;
+import me.grishka.appkit.imageloader.ImageLoaderRecyclerAdapter;
+import me.grishka.appkit.imageloader.ImageLoaderViewHolder;
+import me.grishka.appkit.utils.BindableViewHolder;
+import me.grishka.appkit.views.UsableRecyclerView;
+import me.grishka.houseclub.R;
+import me.grishka.houseclub.api.ClubhouseSession;
+import me.grishka.houseclub.api.model.FullUser;
+
+public abstract class UserListFragment extends BaseRecyclerFragment<FullUser>{
+
+	private int selfID=Integer.parseInt(ClubhouseSession.userID);
+	private UserListAdapter adapter;
+
+	public UserListFragment(){
+		super(50);
+	}
+
+	@Override
+	public void onAttach(Activity activity){
+		super.onAttach(activity);
+		loadData();
+	}
+
+	@Override
+	protected RecyclerView.Adapter getAdapter(){
+		if(adapter==null){
+			adapter=new UserListAdapter();
+		}
+		return adapter;
+	}
+
+	@Override
+	public void onViewCreated(View view, Bundle savedInstanceState){
+		super.onViewCreated(view, savedInstanceState);
+		getToolbar().setElevation(0);
+	}
+
+	@Override
+	public void onConfigurationChanged(Configuration newConfig){
+		super.onConfigurationChanged(newConfig);
+		getToolbar().setElevation(0);
+	}
+
+	private class UserListAdapter extends RecyclerView.Adapter<UserViewHolder> implements ImageLoaderRecyclerAdapter{
+
+		@NonNull
+		@Override
+		public UserViewHolder onCreateViewHolder(@NonNull ViewGroup parent, int viewType){
+			return new UserViewHolder();
+		}
+
+		@Override
+		public void onBindViewHolder(@NonNull UserViewHolder holder, int position){
+			holder.bind(data.get(position));
+		}
+
+		@Override
+		public int getItemCount(){
+			return data.size();
+		}
+
+		@Override
+		public int getImageCountForItem(int position){
+			return data.get(position).photoUrl!=null ? 1 : 0;
+		}
+
+		@Override
+		public String getImageURL(int position, int image){
+			return data.get(position).photoUrl !=null ? data.get(position).photoUrl : "https://upload.wikimedia.org/wikipedia/commons/7/70/User_icon_BLACK-01.png" ;
```

Comment:
If the photoUrl is nulll return a default image , I picked any image though from the internet